workbench-core/swb-common-ui/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+
+module.exports = {
+  extends: ['@aws/eslint-config-workbench-core-eslint-custom', 'next/core-web-vitals'],
+  parserOptions: { tsconfigRootDir: __dirname },
+  rules: {
+    'import/default': 0,
+    'import/namespace': 0,
+    'import/no-named-as-default': 0,
+    'import/no-named-as-default-member': 0,
+    '@next/next/no-html-link-for-pages': 0
```

Comment:
why are we turning these rules off?

Reply:
That's an @insignias question

Reply:
first 4 are necessary because the `@cloudscape-design/components` package doesnt include and Props interfaces in the `index.js` files so eslint complains
last one is because Next expects a `pages` folder, which we purposefully dont have

Reply:
Heard back from @insignias and we can confirm that these are required since these rules don't go well with React/NextJS projects and give out false positive errors (tried removing and checked - got 200+ false positive errors)

Reply:
I confirm these rules are required, this change is based on the accepted answer here: https://stackoverflow.com/questions/44437203/how-do-i-resolve-eslint-import-no-named-as-default